api/crudFunctions/weeklyPicks.tsx

```diff
@@ -0,0 +1,36 @@
+//globally accessible env variable
+declare global {
```

Comment:
Remove this for now and hard code the variables. We'll need to figure out how to do it the proper way with the server after MVP.